### DISCUSSION TITLE
Where should we look for help organizing our project web site

### DISCUSSION THREAD

**Discussion by masinter** [google/docsy#1499] [General]
Every time I sit down to edit our Hugo / Docsy project web site I get a kind of writer's block -- too many constraints to optimize the web site user experience.  We chose Hugo/docsy as the platform, because a big portion of what we will host is documentation. But some things don't translate -- we have a node.js server, and documents on Google Drive, and a bibliography in Zotero and a youtube channel ...

Docsy has oodles of features, but it seems harder to wrangle the overall organization than GitBook or Wiki or even raw HTML + Bootstrap (the alternatives we've tried).
 
Our project is to revive some old software, https://interlisp.org. That URL will give you details about the project, and there's a youtube video of a talk about the project which will be linked from the project web site once I've figured out where the link should go.

I've tried pursuing various suggestions about how to optimize navigation, and every time I freeze and go work on something elsee. https://github.com/orgs/Interlisp/projects/5 will show you the level of advice.

I'm looking for someone to help with the organization. I did try to post a job on UpWork but I've had trouble finding a match. SEO and creative copy aren't so important. Clarity, ease of navigation, perhaps a sense of geeky fun?

Is anyone using freelancers to help maintain their web?  Where did you find them? Or if it's you.
Are there people willing to donate their time to a non-profit (501c3) group, or take on some part for a nominal fee?
 -- https://LarryMasinter.net LMM@interlisp.org

**Comment by fekete-robert** [comment]
Hi, I regularly see similar Hugo-related requests on the Hugo forums, though those are usually more related to the web-development aspect of the sites: https://discourse.gohugo.io/
